packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsCompletedRow/components/TransactionsCompletedRowAccountDetails/TransactionsCompletedRowAccountDetails.tsx

```diff
@@ -52,7 +54,7 @@ const TransactionsCompletedRowAccountDetails: React.FC<TProps> = ({
                     <WalletText color='general' size='xs' weight='bold'>
                         {displayAccountName}
                     </WalletText>
-                    {!isDemo && <WalletListCardBadge label={landingCompanyName} />}
+                    {!isDemo && <WalletListCardBadge label={mt5LandingCompanyName ?? landingCompanyName} />}
```

Comment:
not sur if thats possible - but should we explicitly check for account type instead of relying on presence of certain labels?

Reply:
actually, on the second thought: is there even need for second prop within that component? 
seems that actually parent component is responsible for checking the account type anyway, so it could just provide one landingCompanyName to TransactionsCompletedRowAccountDetails

Reply:
These two are of different types, `landingCompanyName`'s being narrower and passed to a function within this component which requires this type, so we cannot change it.
`mt5LandingCompanyName` has a wider type, so I thought I couldn't join these